Azure/azure-cli
# [AKS] `az aks nodepool add`: Support the new SKU Mariner for parameter `--os-sku`

**Related command**
az aks
az aks create
az aks nodepool add

**Description**
Add support for new os sku "Mariner". Core CLI version of https://github.com/Azure/azure-cli-extensions/pull/5276 / https://github.com/Azure/azure-cli-extensions/pull/5174

**Testing Guide**
az aks nodepool add --os-sku Mariner 
az aks nodepool show -> osSku should be Mariner

same for az aks create/show

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.
